cmd/tiller/tiller.go

```diff
@@ -97,7 +97,7 @@ func main() {
 	flag.Parse()
 
 	if *printVersion {
-		fmt.Println(version.GetVersion())
+		fmt.Println(version.FormatVersion(version.GetVersionProto(), true))
```

Comment:
With the old `version.GetVersion` function when there was no `BuildMetadata` it would just share the version number (e.g., `1.2.3`). In the new setup it will always share the `GitCommit`. Even for releases (as opposed to dev versions). Why the change?